tools/postman/AWS production.postman_environment.json

```diff
@@ -0,0 +1,29 @@
+{
+	"id": "40e90b8a-5aef-47ce-b53b-b0142ba1ceb9",
+	"name": "AWS production",
+	"values": [
+		{
+			"key": "baseURL",
+			"value": "https://eapd-api.cms.gov",
+			"enabled": true
+		},
+		{
+			"key": "created-api-id",
+			"value": "1",
+			"enabled": true
+		},
+		{
+			"key": "auth-username",
+			"value": "TxTest",
+			"enabled": true
+		},
+		{
+			"key": "auth-password",
+			"value": "password",
+			"enabled": true
+		}
```

Comment:
I'm pretty sure I know the answer, but just to double-check, these are not the actual username and password for this account, right?

Reply:
Yes, that's right, they're not the actual passwords.  I just put "password" everywhere but I wanted to have the variable in there so you wouldn't have to remember what it was called.